payments/utils.py

```diff
@@ -25,7 +25,7 @@ def _get_last_payments():
     client = FioBank(token=settings.FIO_BANK_TOKEN)
 
     today = timezone.now()
-    date_from = (today - timedelta(days=3)).strftime(DATE_FORMAT)
+    date_from = (today - timedelta(days=settings.FIO_BANK_PROCESS_DAYS)).strftime(DATE_FORMAT)
```

Comment:
The tests will have to be refactored as well. 3 was a hard-coded constant, so if you change it they will fail. Sorry for that.